.travis.yml

```
@@ -35,7 +35,10 @@ script:
   - flake8 --max-line-length=115  # Enforce code style (but relax line length limit a bit).
   - set -e
   - make -C docs html  # Build the documentation.
-  # Pubish the documentation to GitHub Pages.
-  - pip install doctr
-  - doctr deploy --built-docs docs/build/html --deploy-repo NSLS-II/NSLS-II.github.io event-model
-  - doctr deploy --built-docs docs/build/html --deploy-repo bluesky/bluesky.github.io event-model
+  - |
+    if [ $PUBLISH_DOCS ] then
```

Thanks. I think this is missing:
```suggestion
    if [ $PUBLISH_DOCS ]; then
```

---

Sorry I wasn't precise about the location of the semicolons. It's still should be a part of `if ... then`.